Lib/test/_test_multiprocessing.py

```diff
@@ -269,6 +269,62 @@ def _test(cls, q, *args, **kwds):
             q.put(bytes(current.authkey))
             q.put(current.pid)
 
+    def test_parent_process_attributes(self):
+        if self.TYPE == "threads":
+            self.skipTest('test not appropriate for {}'.format(self.TYPE))
+        from multiprocessing.process import current_process
+        rconn, wconn = self.Pipe(duplex=False)
+        p = self.Process(target=self._test_send_parent_process, args=(wconn,))
+        p.start()
+        p.join()
+        parent_process = rconn.recv()
+        assert parent_process.pid == current_process().pid
+        assert parent_process.name == current_process().name
+
+    @classmethod
+    def _test_send_parent_process(cls, wconn):
+        from multiprocessing.process import parent_process
+        wconn.send(parent_process())
```

Comment:
It seems a little weird to return this as it might try to return the `_sentinel` object. Why not return just the necessary info to be sure to avoid pickling issue?